.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
I will remove this file

Reply:
this file is used by several code editors. I put it into project because of inconsistence of line-breaks (Windows uses CRLF and *nix uses LF). This file fix environment differences (in code editors like vscode, which I'm using.). If you don't think it's necessary, that's ok to remove.

Reply:
You convinced me. I leave it